src/helpers/getErrorString.ts

```diff
@@ -0,0 +1,169 @@
+import { Horizon } from "stellar-sdk";
```

Comment:
Copied this file from AV. We use this a lot, maybe it's worth adding to frontend helpers.